.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -19,11 +19,5 @@ assignees: ''
 3. Scroll down to '....'
 4. See error
 
-## Additional Context
-- Add any other context about the problem here.
-
-## Doc Changes
-- Confirm if needed & identify which repos and docs are impacted.
-
-## Screenshots
-If applicable, add screenshots to help explain your problem.
+## Additional Context (if applicable)
+- _(Add any other context about the problem here; for example: doc changes, screenshots, logs, links, etc)_
```

Comment:
Can we take the (if applicable) off of the title and put in the line below? If I want to use this, I have to delete on two lines vs. just highlighting and typing